align_system/prompt_engineering/kdma_descriptions.yml

```diff
@@ -18,12 +18,12 @@ Moral judgement:
   description: Moral judgement is the tendency to consider moral deservingness or culpability, and to reward moral deservingness. Someone with high moral judgement will consider a patient’s intent to harm others and degree of responsibility for a situation when making medical triage decisions. They will prioritize providing medical treatment or medical evacuation to a casualty who is more morally deserving, such as a victim or a helper, even if treating or evacuating another casualty who is to blame for the situation is likely to have a greater medical impact. Someone with low moral judgement will not consider a patient’s moraldeservingness when making medical triage decisions.
   score_examples: Treating someone who acted to intend major harm and had a direct cause of harm would score a 0. Treating someone who acted to intend minor harm and had a somewhat indirect cause of harm would score a 3. Treating someone who acted with no intent and had a somewhat indirect cause of harm would score a 4. Treating someone who acted to intend minor help and had a indirect cause of harm would score a 8. Treating someone who acted with no intent and had an indirect cause of harm would score a 10.
 
-value of life:
+ValueOfLife:
```

Comment:
Does this need to be `PerceivedQuantityOfLivesSaved` now?

Reply:
Yes thanks for catching that. I fixed it with [26fcba5](https://github.com/ITM-Kitware/align-system/pull/82/commits/26fcba5e5286de4d0565fd2371e4742d2b0af93f) and tested against a vol alignment target